### DISCUSSION TITLE
Call for ideas for 1.0

### DISCUSSION THREAD

**Discussion by tbolt** [tbolt/boltcss#28] [Ideas]
I'm pretty happy with where Boltcss is at but im open to ideas on what would "round out" this project for a 1.0 release.

**Comment by Nandaleio** [comment]
Could `<dialog>` be part of the 1.0 ? for now I image we can just add a `--border-radius` to the dialog element.
Also idk when the 1.0 is be planned to be release, but maybe it could be interesting to do something about the future `<selectedcontent>` element.

**Comment by tbolt** [comment]
@Nandaleio I like that idea. Feel free to make a PR if you would like. Otherwise I will likely work on it in a few weeks.

**Comment by Nandaleio** [comment]
What about cards using the `article` element ? (maybe support header and footer inside the article ?)

Would tooltip/popover fall in the scope ?

I also thought about invalid input styling and a loading spinner but all that would require to add extra attribute to the element.
A bit like what pico css does :

`<input type="text" aria-invalid="true">`
&
`<article aria-busy="true"></article>`

I don't know if that would get to far from the original idea of this project.
What do you guys think ?

**Reply by tbolt** [reply]
These are great ideas. I experimented with tooltip/popover a bit but decided to punt on that for now, I'm open to revisiting it though.

I think article doesn't really need much styling and I wouldn't want it to be specific for a "card" style. I think of article more as a semantic tag that doesn't necessarily have style. So I don't think that should be included.

I'll think about the invalid input styling and take a look at pico css.
